allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/AllureCucumberJvm.java

```diff
@@ -250,12 +250,12 @@ private void createDataTableAttachment(final List<DataTableRow> dataTableRows) {
 
     @Override
     public void embedding(final String string, final byte[] bytes) {
-        //Nothing to do with Allure
+        Allure.addAttachment("Screenshot", new ByteArrayInputStream(bytes));
```

Comment:
use `lifecycle.addAttachment(...)` instead

Reply:
How should I mock the byte array? Should it be stored as an image in resources or hardcoded value or something else?

Reply:
You can simply use something like `"Test string".getBytes(UTF_8)`